DietPi-Software | Python3

Joulinar opened the pull request:
Looks like there was/is a change to download `get-pip.py` on older Debian version. At least on Stretch this is failing atm as content inside `get-pip.py` change into a message. 

```
#!/usr/bin/env python
from __future__ import print_function
import sys
import textwrap

message = """
Hi there!

The URL you are using to fetch this script has changed, and this one will no
longer work. Please use get-pip.py from the following URL instead:

    https://bootstrap.pypa.io/pip/3.5/get-pip.py

Sorry if this change causes any inconvenience for you!

We don't have a good mechanism to make more gradual changes here, and this
renaming is a part of an effort to make it easier to us to update these
scripts, when there's a pip release. It's also essential for improving how we
handle the `get-pip.py` scripts, when pip drops support for a Python minor
version.

There are no more renames/URL changes planned, and we don't expect that a need
would arise to do this again in the near future.

Thanks for understanding!

- Pradyun, on behalf of the volunteers who maintain pip.
"""

print(message, file=sys.stderr)
sys.exit(1)
```

Even if Buster is still working, I adjusted the URL as well. 

**Status**: Ready
- [x] Update Pip3 install

**Reference**: https://dietpi.com/phpbb/viewtopic.php?f=9&t=8968

**Commit list/description**:
- DietPi-Software | Python3 - update download URL to reflect recent changes

Joulinar committed:
v7.2

DietPi-Software | Python3 - update download URL to reflect recent changes

MichaIng commented:
I was just confused as I thought it's about the dedicated Python 3.5 installer, but it's about a new sub directory. The default installer is still also located at the original location and docs haven't changed (yet): https://pip.pypa.io/en/stable/installing/#installing-with-get-pip-py

But both match, so it's probably a good idea to change to the sub dir in both cases.

MichaIng committed:
v7.2

+ CHANGELOG | Python 3: Resolved an issue where installing pip on Stretch systems failed, due to a changed download URL

Joulinar commented:
> But both match, so it's probably a good idea to change to the sub dir in both cases.

yes that was my intention to get the new sub dir in both cases. Not sure if they will change something in future with default installer as well

MichaIng committed:
Merge branch 'dev' into python3
